presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControlMetadata.java

```diff
@@ -198,4 +198,40 @@ private static boolean hasAdminRole(Set<HivePrincipal> roles)
     {
         return roles.stream().anyMatch(principal -> principal.getName().equalsIgnoreCase(ADMIN_ROLE_NAME));
     }
+
+    @Override
+    public void dropSchemaPrivileges(ConnectorSession session, String schemaName)
+    {
+        // nothing needs to be done here because drop schema in metastore will drop its privileges
+    }
+
+    @Override
+    public void dropTablePrivileges(ConnectorSession session, SchemaTableName tableName)
+    {
+        // nothing needs to be done here because drop table in metastore will drop its privileges
```

Comment:
does this call need to exist then?

Reply:
Yes, otherwise it will throw an exception.